documentation/memory/step3/using_gc_traces.md

```diff
@@ -127,63 +332,14 @@ PerformanceEntry {
 }
 ```
 
+| Property   | Interpretation                                                                                  |
+|------------|-------------------------------------------------------------------------------------------------|
+| name       | The name of the performance entry.                                                              |
+| entryType  | The type of the performance entry.                                                              |
+| startTime  | The high-resolution millisecond timestamp is marking the starting time of the Performance Entry.|
+| duration   | The total number of milliseconds elapsed for this entry.                                        |
+| kind       | The type of garbage collection operation that occurred.                                         |
+| flags      | The high-resolution millisecond timestamp is marking the starting time of the Performance Entry.|
 
-<table>
-  <tr>
-    <th>Property</th>
-    <th>Interpretation</th>
-  </tr>
-  <tr>
-    <td>name</td>
-    <td>The name of the performance entry.</td>
-  </tr>
-  <tr>
-    <td>entryType</td>
-    <td>The type of the performance entry. </td>
-  </tr>
-  <tr>
-    <td>startTime</td>
-    <td>The high resolution millisecond timestamp <br/> marking the starting time of the Performance Entry.</td>
-  </tr>
-  <tr>
-    <td>duration</td>
-    <td>The total number of milliseconds elapsed for this entry. </td>
-  </tr>
-  <tr>
-    <td>kind</td>
-    <td>The type of garbage collection operation that occurred.</td>
-  </tr>
-  <tr>
-    <td>flags</td>
-    <td>Additional information about garbage collection operation.</td>
-  </tr>
-</table>
-
-For more information, you can refer to [the documentation about performance hooks](https://nodejs.org/api/perf_hooks.html).
-
-## Examples of diagnosing memory issues with trace option:
```

Comment:
Couldn't we use it somehow? The content is pretty good.

Reply:
I use them in the body of the guide.